vanilla-installer/main.py

```diff
@@ -73,26 +98,41 @@ def text_update(text: str, widget=None, mode: str = "info") -> None:
         if mode == "fg":
             logging.debug(text)
         if mode == "warn":
-            logging.warn(text)
+            logging.warning(text)
         if mode == "error":
             logging.error(text)
         if mode == "success":
             logging.info(text)
 
 
 def command(text: str) -> str:
-    output = logging.debug(subprocess.check_output(text.split()).decode("utf-8"))
+    """Runs a command with subprocess.
+
+    Returns:
+        str: The output of the command.
+    """
+    output = str(logging.debug(subprocess.check_output(text.split()).decode("utf-8")))
     text_update(output, mode="fg")
     return output
 
 
+# This function isn't ever called, it's probably better to depreciate it (feedback wanted)
```

Comment:
Anyone have thoughts here?

Reply:
Well, seems pretty useless, at least in current state... 
You should rewrite it to something more useful than just opening web page, or remove it completely since it's not used anywhere anyway

Reply:
Removed.